packages/frontend/src/common/components/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+interface Params {
+  title: string
+}
+
+const Button = ({ title }: Params) => {
+  return (
+    <button className="btn bg-accent h-[48px] min-w-[192px] text-white normal-case hover:bg-accentHover">
```

Comment:
```suggestion
    <button className="btn bg-accent h-12 min-w-48 text-white normal-case hover:bg-accentHover">
```

Reply:
I've changed `h-[48px]` to `h-12` but looks like there is no such think as `min-w-48`